core/tee/tee_svc_storage.c

```diff
@@ -25,19 +25,46 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <tee/tee_svc_storage.h>
-
 #include <kernel/tee_ta_manager.h>
-#include <tee_api_defines.h>
 #include <kernel/tee_misc.h>
+#include <mm/tee_mmu.h>
 #include <tee/tee_fs.h>
 #include <tee/tee_fs_defs.h>
 #include <tee/tee_obj.h>
 #include <tee/tee_svc.h>
-#include <mm/tee_mmu.h>
 #include <tee/tee_pobj.h>
+#include <tee/tee_svc_storage.h>
+#include <tee_api_defines.h>
+#include <tee_api_defines_extensions.h>
 #include <trace.h>
 
+static const struct tee_file_operations *file_ops(uint32_t storage_id)
+{
+
+	switch (storage_id) {
+	case TEE_STORAGE_PRIVATE:
+#ifdef CFG_REE_FS
+		return &ree_fs_ops;
+#else
+#ifdef CFG_RPMB_FS
```

Comment:
This could be written as `#elif defined(CFG_RPMB_FS)` instead to make it a bit less complicated.